pkg/cmdline/cmdline.go

```diff
@@ -156,14 +157,21 @@ func progress(cli *cmdline, app, desc string, codes <-chan builder.SummaryStatus
 	}
 }
 
-func passStr(msg string) string {
-	const pass = "SUCCESS " + "⚓"
-	return fmt.Sprintf("%s: %s", green(msg), pass)
+func passStr(msg string, disableEmoji bool) string {
+	return fmt.Sprintf("%s: %s", green(msg), concatStrAndEmoji("SUCCESS", " ⚓ ", disableEmoji))
 }
 
-func failStr(msg string) string {
-	const fail = "FAIL " + "❌"
-	return fmt.Sprintf("%s: %s", red(msg), fail)
+func failStr(msg string, disableEmoji bool) string {
+	return fmt.Sprintf("%s: %s", red(msg), concatStrAndEmoji("FAIL", " ❌ ", disableEmoji))
+}
+
+func concatStrAndEmoji(text string, emoji string, disableEmoji bool) string {
+	var concatStr bytes.Buffer
+	concatStr.WriteString(text)
+	if !disableEmoji {
```

Comment:
I find the use of `disableEmoji` throughout a bit hard to follow, and this double negative is I think why - there is an extra cognitive load of 'and pass whether we're _not_ showing emoji' and 'if we're **not** not showing emoji...".  Consider a positive name `displayEmoji` and a positive condition check.

Reply:
Agree to it 👍  changed it